src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs

```diff
@@ -268,21 +268,26 @@ public BubbleNullSchema()
         var query = new ObjectGraphType();
 
         query.Field<NonNullGraphType<DataGraphType>>("nonNullableDataGraph")
-            .Resolve(c => new DataGraphType { Data = c.Source as Data }
+            .Resolve(c => new DataModel { Data = c.Source as Data }
         );
 
         query.Field<DataGraphType>("nullableDataGraph")
-            .Resolve(c => new DataGraphType { Data = c.Source as Data }
+            .Resolve(c => new DataModel { Data = c.Source as Data }
         );
 
         query.Field<NonNullGraphType<ListGraphType<NonNullGraphType<DataGraphType>>>>("nonNullableListOfNonNullableDataGraph")
-            .Resolve(_ => new[] { new DataGraphType() });
+            .Resolve(_ => new[] { new DataModel() });
 
         Query = query;
     }
 }
 
-public class DataGraphType : ObjectGraphType<DataGraphType>
```

Comment:
Ha)

Reply:
That means that in theory in very specific cases someone **may** want to inherit from `ObjectGraphType<OtherGraphType>`.